src/meshapi/views/forms.py

```diff
@@ -105,6 +113,21 @@ def join_form(request: Request) -> Response:
 
 
 def process_join_form(r: JoinFormRequest, request: Optional[Request] = None) -> Response:
```

Comment:
Does this belong here, maybe it should it be included in `join_form()` above so that we don't need to worry about this validation in the join form replay CLI?